containers/Dockerfile_base-focal

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Is there any convention for where people keep their docker files?

If not I think we should move to `Tools/setup/Dockerfile` (no base or focal).